### PR TITLE
Created initial version of the calculator app

### PR DESCRIPTION
**Pull request by SwathiPottigari**
@codingdawg

Please review the changes
1. Created the calculator app that runs on the browser.
2.Created a separate JS file.
3.The app does the addition,multiplication and division operations .
4.It also remembers the history.

### CODE REVIEW

**Review by codingdawg — APPROVED** [review]
LGTM!

**Comment by codingdawg** [comment]
@addprs please review

File: CalculatorApp.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    else {
+        b = storage;
+        display = display +"+"+ a;
+        storage = storage + a;
+        resultelement.innerHTML = display + "</br>" +"<b>" + storage +"</b>";
+    }
+    
+}
+
+function mulNumbers() {
```
Comment (codingdawg) [inline CalculatorApp.js]: Use a detailed method name `multiplyNumbers`

File: CalculatorApp.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
+// JavaScript source code
+var storage = 0;
+var display = "";
+var resultelement;
+
+function initializeGloabls() {
```
Comment (codingdawg) [inline CalculatorApp.js]: initializeGloabls method is spelled incorrectly.

**Review by codingdawg — CHANGES_REQUESTED** [review]

File: CalculatorApp.js (hunk 1 above)
Comment (SwathiPottigari) [reply CalculatorApp.js]: Renamed the method name

File: CalculatorApp.js (hunk 2 above)
Comment (SwathiPottigari) [reply CalculatorApp.js]: corrected the method name

**Pull request merged**